### DISCUSSION TITLE
Can other people see my as a mob ?

### DISCUSSION THREAD

**Discussion by 0nlyRich** [Draylar/identity#310] [Q&A]
Can my friends on my server see me as a mob to ?

**Comment by Draylar** [comment]
Forge version currently has a bug where you can't see the Identity of other people, but I will fix this soon.

edit: but just to clarify, your friends will need to install Identity on their clients as well. Identity isn't a server-only mod.

**Reply by 0nlyRich** [reply]
I know they already have it but thank you ^^ when will they see me as a mob ? Like in how many weeks
